libpod/boltdb_state_internal.go

```diff
@@ -805,6 +832,23 @@ func (s *BoltState) removeContainer(ctr *Container, pod *Pod, tx *bolt.Tx) error
 		return errors.Wrapf(define.ErrCtrExists, "container %s is a dependency of the following containers: %s", ctr.ID(), strings.Join(deps, ", "))
 	}
 
+	// Does the container have exec sessions?
+	ctrExecBkt := ctrExists.Bucket(execBkt)
+	if ctrExecBkt != nil {
+		sessions := []string{}
+		err = ctrExecBkt.ForEach(func(id, unused []byte) error {
+			sessions = append(sessions, string(id))
+
+			return nil
+		})
+		if err != nil {
+			return err
+		}
+		if len(sessions) != 0 {
+			return errors.Wrapf(define.ErrCtrExists, "container %s has active exec sessions: %s", ctr.ID(), strings.Join(sessions, ", "))
+		}
+	}
+
```

Comment:
this block and the previous block at 800 appear to be the same and also have the same error message.  any chance we could stub that out?

Reply:
Actually they're completely redundant. Oops.